### PR TITLE
Adding the recommended version for policy generator

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/25740#issuecomment-1279102380

### CODE REVIEW

**Review by oafischer — APPROVED** [review]
lgtm

**Comment by dockerymick** [comment]
@oafischer Thanks so much for your review!

**Comment by dockerymick** [comment]
Keeping this PR open until @JustinKuli updates the policy generator version to 1.10.0. I currently have it set to 1.9.1

File: governance/policy_generator.adoc [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 The policy generator is a part of the {product-title} application lifecycle subscription GitOps workflow that generates {product-title-short} policies using Kustomize. The policy generator builds {product-title-short} policies from Kubernetes manifest YAML files, which are provided through a `PolicyGenerator` manifest YAML file that is used to configure it. The policy generator is implemented as a Kustomize generator plugin. For more information on Kustomize, see the link:https://kustomize.io/[Kustomize documentation]. 
 
+The policy generator version bundled in this version of {product-title-short} is v1.9.1.
```
Comment (JustinKuli) [inline governance/policy_generator.adoc]: Policy Generator in 2.7 is now `v1.10.0`

Comment (dockerymick) [reply governance/policy_generator.adoc]: Thanks for confirming Justin!

**Pull request merged**